src/controller/cve.controller/index.js

```diff
@@ -512,7 +488,7 @@ router.post('/cve/:id/reject',
     }
   }
   #swagger.responses[200] = {
-    description: 'Returns the CVE record updated.',
+    description: 'Returns the CVE record updated',
     content: {
       "application/json": {
         schema: { $ref: '/schemas/cve/create-cve-record-rejection-response.json' }
```

Comment:
response body should match this
{
    "message": "CVE-2008-0006 record was successfully submitted.",
    "created": {
        "dataType": "CVE_RECORD",
        "dataVersion": "5.0",
        "cveMetadata": {
            "cveId": "CVE-2008-0006",
            "assignerOrgId": "b8e70571-7ecc-4e74-b5bb-cd168f4198f7",
            "state": "REJECTED",
            "dateReserved": "2013-12-25T00:00:00.000Z",
            "dateUpdated": "2022-05-12T17:18:57.510Z",
            "dateRejected": "2022-05-12T17:18:57.510Z",
            "assignerShortName": "mitre"
        },
        "containers": {
            "cna": {
                "providerMetadata": {
                    "orgId": "b8e70571-7ecc-4e74-b5bb-cd168f4198f7",
                    "shortName": "mitre",
                    "dateUpdated": "2022-05-12T17:18:57.510Z"
                },
                "rejectedReasons": [
                    {
                        "lang": "en-Gb",
                        "value": "I professional site herself recently behavior. Situation institution meeting recognize successful.",
                        "supportingMedia": [
                            {
                                "type": "test/markdown",
                                "base64": false,
                                "value": "*this* _is_ supporting media in ~markdown~"
                            }
                        ]
                    }
                ],
                "replacedBy": [
                    "CVE-1999-0006"
                ]
            }
        }
    }
}